src/functions.h

```diff
@@ -1,10 +1,10 @@
-extern const void hello_c();       // lib.c
-extern const void hello_cpp();     // lib.cpp
-extern const void hello_d();       // lib.d
-extern const void hello_fortran(); // lib.f95
-extern const void hello_rust();    // lib.rs
-extern const void hello_zig();     // lib.zig
-extern const void hello_carp();    // lib.carp
+extern const void hello_c();
+extern const void hello_cpp();
+extern const void hello_carp();
```

Comment:
Doesn't carp comes before cpp? Cause `ca` comes before `cp`

Reply:
It's technically C++ which is why I put it before Carp.